src/main/java/seedu/address/model/ResumeBook.java

```diff
@@ -225,7 +226,14 @@ public void deleteInternship(Internship key) {
         internships.remove(key);
         for (Item item : resumes) {
             Resume resume = (Resume) item;
-            resume.getInternshipIds().remove(Integer.valueOf(id));
+            Resume newresume = new Resume(resume.getName(), resume.getId(), resume.getTags());
```

Comment:
This should be camel case: `newResume`. 
```suggestion
            Resume newResume = new Resume(resume.getName(), resume.getId(), resume.getTags());
```